src/app/language-support/page.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import { getAllLanguageSupport } from '@/api/supabase/queries/language-support';
+import { useEffect, useState } from 'react';
+import { LanguageSupport } from '@/types/schema';
+
+export default function Page() {
+  const [languageSupport, setLanguageSupport] = useState<LanguageSupport[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const data = await getAllLanguageSupport();
+        setLanguageSupport(data as LanguageSupport[]);
+      } catch (error) {
+        console.error('(useEffect)[LanguageSupport]', error);
+      }
+    };
+    fetchData();
+  }, []);
+
+  const getIsRemoteValue = (isRemote: boolean | undefined) => {
+    let isRemoteDisplay;
+    switch (isRemote) {
+      case true:
+        isRemoteDisplay = 'true';
+        break;
+      case false:
+        isRemoteDisplay = 'false';
+        break;
+      default:
+        isRemoteDisplay = 'null';
+    }
+    return isRemoteDisplay;
+  };
```

Comment:
we can simplify this to a bunch of if and returns.
i.e.
```
if (isRemote) return 'true';
if (isRemote === false) return 'false';
return 'null';
```

Reply:
sg! changed